lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +139,15 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($type === \PDO::PARAM_LOB || -99 === $type) {
```

Comment:
@soyuka `PDO::PARAM_LOB` is only used in `BinaryType` and `BlobType` in DBAL so yeah it always corresponds to binary values in the core. I also guess that DBAL 2.x had in mind to map `PDO::PARAM_LOB` only to binary types (given that even `TextType`, `ObjectType` etc. map to `PDO::PARAM_STR`).
So I think we can assume safely assume `PDO::PARAM_LOB` always to be bound to binary.